### PR TITLE
Become kedro-community

### PR DESCRIPTION
**Pull request by laisbsc**
## Description
Changing the repository name to Kedro-community.

## Development notes
Description on the README and repository name.

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Assigned myself to the PR
- [ ] Added tests to cover my changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking “Submit Pull Request”:

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,9 @@ This repo contains links to all official and community created example Kedro pro
   - [Kedro 0.15.5](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.5)
   - [Kedro 0.15.8](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.8)
   - [Kedro 0.16.1](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.1)
-  - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)
+  - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)
```
Comment (921kiyo) [inline README.md:14]:
```suggestion
  - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)
```

**Comment by laisbsc** [comment]
I am not sure on how to change the name of the repo. Help, please. 😸 

And, let me work on a different structure for the community examples on the `README`. I've pushed those changes too early, sorry about that.

**Comment by MerelTheisenQB** [comment]
> I am not sure on how to change the name of the repo. Help, please. 😸

@laisbsc I think only admins of the repo can change the name. So I guess @yetudada or @idanov ?

**Comment by laisbsc** [comment]
> @laisbsc I think only admins of the repo can change the name. So I guess @yetudada or @idanov ?

@MerelTheisenQB Kiyo just told me 😅 Thank you for the help.

I am also thinking about changing the way we structure the `README`, giving more emphasis to the community projects. @stichbury suggestions?

There is another sprint happening soon and this is the easiest way to engage newbies with Kedro, ask them to create an example with different datasets :)

File: README.md [hunk 2]
```diff
@@ -1,24 +1,46 @@
...
-## Where can I find more learning material and `kedro-examples`?
-
-The [`Kedro-Training`](https://github.com/quantumblacklabs/kedro-training) repository contains documentation that you can use to deliver Kedro training.
-
-These repositories contain the official Kedro tutorial projects, also known as starters:
+# What is `kedro-community`?
+
+This repo contains links to articles, podcasts and videos about Kedro, as well as, community created examples of Kedro projects. If you're looking to engage with the community then we suggest checking out the [Kedro Community Discourse](https://discourse.kedro.community/) channel.
+
+## Where can I find articles, podcasts and videos about Kedro?
```
Comment (921kiyo) [inline README.md:5]: This section is quite long, so people might not notice there's a section "Where can I find more example projects using Kedro?" underneath. Maybe we can swap these two sections?

**Review by 921kiyo — APPROVED** [review]
👍

File: README.md [hunk 3]
```diff
@@ -1,23 +1,46 @@
...
-  - [Kedro 0.15.8](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.8)
-  - [Kedro 0.16.1](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.1)
-  - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)
-
-## Where can I find more learning material and `kedro-community`?
-
-The [`Kedro-Training`](https://github.com/quantumblacklabs/kedro-training) repository contains documentation that you can use to deliver Kedro training.
-
-These repositories contain the official Kedro tutorial projects, also known as starters:
+This repo contains links to articles, podcasts and videos about Kedro, as well as, community created examples of Kedro projects. If you're looking to engage with the community then we suggest checking out the [Kedro Community Discourse](https://discourse.kedro.community/) channel.
```
Comment (laisbsc) [inline README.md:3]:
```suggestion
This repo contains links to articles, podcasts and videos about Kedro, as well as, community created examples of Kedro projects. If you're looking to engage with the community then we suggest checking out the [Kedro Community forum](https://discourse.kedro.community/) channel.
```

**Review by laisbsc — COMMENTED** [review]
Will we add new Kedro example projects to the table?

File: README.md (hunk 2 above)
Comment (yetudada) [reply README.md:5]: I like this suggestion, will do!

**Comment by yetudada** [comment]
> Will we add new Kedro example projects to the table?

Well ideally it's up to the community to maintain this 😄

**Pull request merged**